docs/xks/operator-guide/architecture-and-design.md

```diff
@@ -0,0 +1,117 @@
+---
+id: architecture-and-design
+title: Architecture and design
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+## Overview
+
+[Xenit Kubernetes Framework](https://github.com/search?q=topic%3Axenit-kubernetes-framework+org%3AXenitAB+fork%3Atrue) (XKF) is the open source building blocks for a service Xenit AB provides customers: [Xenit Kubernetes Service](https://xenit.se/it-tjanster/kubernetes-eng/) (XKS)
+
+In the perspective of [Microsoft Cloud Adoption Framework](https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/enterprise-scale/architecture) (CAF), Xenit Kubernetes Service is an enterprise-scale landing zone.
+
+The primary goal of the framework is to provide a standardized service to customers where the creators of the framework (employees at Xenit) can manage multiple environments without having to be the one who set it up - because it looks just about the same in all places. This also provides the ability to fix issues, change standards in one place and not having to repeat the same thing over and over at all customer (or two engineers solving the same problem at two different customers).
+
+<img alt="XKS Overview" src={useBaseUrl("img/assets/xks/operator-guide/aks-overview.jpg")} />
+
+## Role-based access management
+
+All role-base access control (RBAC) and identity & access management (IAM) is handled with Azure AD. Azure AD groups are created and nested using the framework and framework admins as well as customer end users are granted access through these different groups.
+
+Everything, where possible, exposes two different permissions: Reader and Contributor
```

Comment:
It is not clear to me what "Everything" refers to. I doubt my sandwich has a reader and a contributor.

Reply:
done!